pallets/messages/src/lib.rs

```diff
@@ -108,7 +108,11 @@ pub mod pallet {
 
 		#[cfg(feature = "runtime-benchmarks")]
 		/// A set of helper functions for benchmarking.
-		type Helper: BenchmarkHelper<Self::AccountId>;
```

Comment:
I was hoping we only needed to have one helper here.

I wonder if we could do something like
```
pub struct BenchmarkHelperThing<SomeMSAThing, SomeSchemaThing>(PhantomData<(SomeMSAThing, SomeSchemaThing)>);

impl<SomeMSAGen, SomeSchemaGen, AccountId> BenchmarkHelper<AccountId> where
where SomeMSAGen:  ...,

{

}
```




```

Reply:
@enddynayn  I went that route but got stuck on an error and it got too complicated at that point, so I think this is simple enough. Your are welcome if can make it work without making it too complicated.

Reply:
Sounds good. Rust is not easy. This is an improvement from what we have now.